tests/shipping-preview/models/Delivery - Same Price Packages.model.js

```diff
@@ -0,0 +1,31 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import { insertCoupon } from '../../../utils/payment-actions'
+import {
+  checkShippingPreviewResult,
+  fillShippingPreviewDelivery,
+} from '../../../utils/shipping-actions'
+import { ACCOUNT_NAMES, SKUS, SLA_IDS } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Delivery - Same Price Packages - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only delivery', () => {
+      setup({ skus: [SKUS.DELIVERY_MULTIPLE_SLA], account })
+
+      const selectors = []
```

Comment:
Since it's only used in the `else` state what do you think of removing it and directly using the array as a parameter like you did in the `if` statement?

Reply:
☝️

Reply:
Nice!